agents/aws_vpc_net/fence_aws_vpc_net.py

```diff
@@ -0,0 +1,782 @@
+        "getopt": "",
+        "longopt": "invert-sg-removal",
+        "help": "--invert-sg-removal              Remove all security groups except the specified one(s).",
+        "shortdesc": "Remove all security groups except specified..",
+        "required": "0",
+        "order": 7,
+    }
+    all_opt["unfence-ignore-restore"] = {
+        "getopt": "",
+        "longopt": "unfence-ignore-restore",
+        "help": "--unfence-ignore-restore              Do not restore security groups from tag when unfencing (off).",
+        "shortdesc": "Remove all security groups except specified..",
+        "required": "0",
+        "order": 8,
+        
+    }
+    all_opt["filter"] = {
+        "getopt": ":",
+        "longopt": "filter",
+        "help": "--filter=[key=value]           Filter (e.g. vpc-id=[vpc-XXYYZZAA])",
+        "shortdesc": "Filter for list-action",
+        "required": "0",
+        "order": 9
+    }
+    all_opt["boto3_debug"] = {
+        "getopt": "b:",
+        "longopt": "boto3_debug",
+        "help": "-b, --boto3_debug=[option]     Boto3 and Botocore library debug logging",
+        "shortdesc": "Boto Lib debug",
+        "required": "0",
+        "default": "False",
+        "order": 10
+    }    
+    all_opt["onfence-poweroff"] = {
+        "getopt": "",
+        "longopt": "onfence-poweroff",
+        "help": "--onfence-poweroff              Power off the machine async upon fence (this is a network fencing agent...)",
+        "shortdesc": "Power off the machine async..",
+        "required": "0",
+        "order": 11
+    }
+
+
+def main():
+    conn = None 
+
+    device_opt = [
+        "no_password", 
+        "region",
+        "access_key", 
+        "secret_key",
+        "secg",
+        "port",
+        "skip_race_check",
+        "invert-sg-removal",
+        "unfence-ignore-restore",
+        "filter",
+        "boto3_debug",
+        "onfence-poweroff"
+]
+
+    atexit.register(atexit_handler)
+    
+    define_new_opts()
+
+    try:
+        processed_input = process_input(device_opt)
+        options = check_input(device_opt, processed_input)
+    except Exception as e:
+        logger.error(f"Failed to process input options: {str(e)}")
+        sys.exit(EC_GENERIC_ERROR)
+
+    run_delay(options)
+
+    docs = {
+        "shortdesc": "Fence agent for AWS (Amazon Web Services) Net",
+        "longdesc": (
+            "fence_aws_vpc is a Network and Power Fencing agent for AWS VPC that works by "
+            "manipulating security groups. It uses the boto3 library to connect to AWS.\n\n"
+            "boto3 can be configured with AWS CLI or by creating ~/.aws/credentials.\n"
+            "For instructions see: https://boto3.readthedocs.io/en/latest/guide/quickstart.html#configuration"
+            " "
+            "NOTE: If --onfence-poweroff is set the agent wont be able to power on the node again, and will have to be powered on manually or with other automation."
```

Comment:
I forgot "it"... ", and it will".

Reply:
Nice.

Let's just removes -- in front of it, as the agents are mostly used with Pacemaker where you they read variables from stdin and instead of CLI arguments.